.github/workflows/health-check.yml

```diff
@@ -8,15 +8,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '10.x', '12.x', '14.x' ]
+        node: [ '12.x', '14.x', '16.x' ]
```

Comment:
i think this still needs to support node 10

Reply:
we currently do releases for node 12, i didn't think we would need to continue supporting node 10 because of that

Reply:
i went and re-reviewed it i've updated and added node 10, thank you!